### DISCUSSION TITLE
What goes into PLUGIN DATA?

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [barseghyanartur/django-fobi#281] [Q&A] [answered]
Fobi installed and trying to build my first form.  Looking through the documentation and cannot find what goes into PLUGIN DATA? when adding elements.

**Comment by barseghyanartur** [comment]
All plugin related data. It's all JSON.

**Reply by michaeloliveraz** [reply]
Ok, that is fine.  How about documentation on what that JSON would have?  How about an example?  Your help is appreciated.

**Reply by michaeloliveraz** [reply]
This shows in the admin pages, but only after creating the form in the Fobi management pages.

use the fobi management pages to create and enter data. /fobi/ gives the Fobi Dashboard.

**Comment by michaeloliveraz** [comment, marked as answer]
use the Fobi Management pages to create forms and enter data.  See http://localhost:8000/fobi/ for the Dashboard.
